src/Elasticsearch/Endpoints/Indices/Reindex.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Elasticsearch\Endpoints\Indices;
+use Elasticsearch\Endpoints\AbstractEndpoint;
+
+/**
+ * Class Analyze
+ *
+ * @category Elasticsearch
+ * @package  Elasticsearch\Endpoints\Indices
+ * @author   Augustin Husson <husson.augustin@gmail.com>
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache2
+ * @link     http://elasticsearch.org
+ */
+class Reindex extends AbstractEndpoint
+{
+
+    /**
+     * @return string[]
+     */
+    protected function getParamWhitelist()
+    {
+        return array();
```

Comment:
Can you add the parameters listed here to your whitelist?  https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/reindex.json#L9